content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -554,6 +555,59 @@ Follow the below steps to import the dashboard in Grafana:
 
 ![Grafana Dashboard](/img/kubermatic/main/monitoring/kubevirt/grafana.png)
 
-## Breaking Changes
+## Migration to KKP 2.22
```

Comment:
please add as well a new section:
````
### KubeVirt upgrade 
````
it's just a place that we don't forget about it :) later I will fill this section after @sankalp-r 's is merged.